pre-edited-cs/data/Stage/ScriptSource/Stream.txt

```diff
@@ -28,7 +28,7 @@
 <UNI0000<CSS<EVE1001
 
 #1001
-<FL+0160<EVE1002
+<EVE1002
```

Comment:
what's the point of this change?

Reply:
Flag 160 triggers the Booster dialogue in Arthur's House. I moved it from after the Ironhead fight to after the Core fight, in accordance with the randomizer logic.